plugin/src/main/groovy/gorm/tools/databinding/MapBinder.groovy

```diff
@@ -1,18 +1,17 @@
 package gorm.tools.databinding
 
-import org.grails.datastore.gorm.GormEntity
-
 /**
  * binds data from a map to a GormEntity. The map can of course be a JSONObject as is common when binding rest resources
  */
 interface MapBinder {
 
     void bind(Object target, Map<String, Object> source, BindAction bindAction)
-
     void bind(Object target, Map<String, Object> source)
 
-    void bindCreate(Object target, Map<String, Object> source)
+    void bind(Object target, Map<String, Object> source, BindAction bindAction, List<String> whiteList, List<String> blackList)
+    void bind(Object target, Map<String, Object> source, List<String> whiteList, List<String> blackList)
```

Comment:
I think we should do this with an optional args map so we can do something more like `mapBinder.bind(user, p, bindAction, include: bindInclude)` or `mapBinder.bind(user, p, exclude:['foo','bar'])`